azext_edge/edge/providers/support/base.py

```diff
@@ -40,13 +48,17 @@ def process_crd(group: str, version: str, kind: str, plural: str, api_moniker: s
         namespace = r["metadata"]["namespace"]
         namespaces.append(namespace)
         name = r["metadata"]["name"]
-        processed.append({"data": r, "zinfo": f"{namespace}/{api_moniker}/{file_prefix}.{version}.{name}.yaml"})
+        processed.append({
+            "data": r,
+            "zinfo": f"{namespace}/{api_moniker}/{child_group}{file_prefix}.{version}.{name}.yaml"
```

Comment:
at this point - why not have this as a template or helper function somewhere? Also make sure that child_group will not need to include the / when the function is called (so you may need to put in a ` if child_group: child_group += "/"